File: src/jsa/baekjoon/silver/Baekjoon_2750.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder(); // 메모리 효율, 속도 빠름
        StringTokenizer st;
        int N = Integer.parseInt(br.readLine());// 배열 크기
        int arr[] = new int[N];
        for (int i = 0; i < N; i++) {
            arr[i] = Integer.parseInt(br.readLine());
        }
        Arrays.sort(arr); // 오름차순 정렬

        for (int num : arr){
            System.out.println(num);
        }
    }
}

File: src/ljh/baekjoon/silver/baekjoon_2750.py
# baekjoon 알고리즘 2750
# 2023.07.23 이주현

num_of_arr = int(input())

arr = list()
for i in range(num_of_arr):
	arr.append(int(input()))

arr.sort()

for num in arr:
    print(num)